.github/workflows/deploy.yml

```
@@ -0,0 +1,50 @@
+
+on:
+    push:
+        tags:
+            - "*"
+jobs:
+    releaseToWPOrg:
+        name: Release to WordPress.org
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)